src/Data.sol

```diff
@@ -72,14 +70,14 @@ contract Data is IData, IndexResolver {
     function redeployIndex() public view onlyOwner {
         require (msg.value >= (_indexDeployValue * 2), DataErrors.value_less_than_required);
         tvm.accept();
-        tvm.rawReserve(msg.value, 1);
+
```

Comment:
Need to use rawReserve
Owner can freez nft if use contract balance

Reply:
You already tvm accept. If you did it smart contract start pay for this message. And you don't need tvm rawReserve